src/java.base/share/classes/java/lang/invoke/StringConcatFactory.java

```diff
@@ -1091,15 +1091,13 @@ private static MethodHandle generate(Lookup lookup, MethodType args, String[] co
             String className = getClassName(lookup.lookupClass());
 
             byte[] classBytes = ClassFile.of().build(ClassDesc.of(className),
-                    new Consumer<ClassBuilder>() {
-                        @Override
-                        public void accept(ClassBuilder clb) {
-                            clb.withFlags(AccessFlag.FINAL, AccessFlag.SUPER, AccessFlag.SYNTHETIC)
-                                .withMethodBody(METHOD_NAME,
-                                        MethodTypeDesc.ofDescriptor(args.toMethodDescriptorString()),
-                                        ClassFile.ACC_FINAL | ClassFile.ACC_PRIVATE | ClassFile.ACC_STATIC,
-                                        generateMethod(constants, args));
-                    }});
+                    (ClassBuilder clb) -> {
```

Comment:
Since this is single-argument and body has only one statement, it can be written as
```java
                        clb -> clb.withFlags(AccessFlag.FINAL, AccessFlag.SUPER, AccessFlag.SYNTHETIC)
                            .withMethodBody(METHOD_NAME,
                                    MethodTypeDesc.ofDescriptor(args.toMethodDescriptorString()),
                                    ClassFile.ACC_FINAL | ClassFile.ACC_PRIVATE | ClassFile.ACC_STATIC,
                                    generateMethod(constants, args));
```